### PR TITLE
crew: Only perform postinstall only if it is defined by package, hide error when `ctrl+c`

### PR DESCRIPTION
**Pull request by supechicken**
### Changes
- Only perform `@pkg.postinstall` and show `post-install` message only if the `self.postinstall` function is defined (used) by the package recipe
- Hide backtrace when interrupted (`ctrl+c`)

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/supechicken/chromebrew.git CREW_TESTING_BRANCH=postinstall_check CREW_TESTING=1 crew update
```

### CODE REVIEW

File: bin/crew [hunk 1]
```diff
@@ -1877,6 +1880,10 @@ def is_command(name)
   return !!!name[/^[-<]/]
 end
 
+trap('INT') do
+  abort 'Interrupted.'.lightred
+end
```
Comment (uberhacker) [inline bin/crew:1885]: Why is this needed?

Comment (supechicken) [reply bin/crew:1885]: This is used to hide the additional backtrace message when `ctrl+c` is pressed (since this is not an error, backtrace messages are unnecessary):
```
# without trap
$ ruby -e 'sleep 10'
^C-e:1:in `sleep': Interrupt
        from -e:1:in `<main>'

# with trap
$ ruby -e 'trap("INT") { abort "This hide the backtrace" }; sleep 10'
^CThis hide the backtrace
```

Comment (uberhacker) [reply bin/crew:1885]: I'm not sure this is a good idea.  There might be an interrupt internally and traceback would be helpful in those cases.

Comment (supechicken) [reply bin/crew:1885]: As I know the interrupt signal is reserved as a user signal (should be fired by user only)

Comment (uberhacker) [reply bin/crew:1885]: Alright then I'll just pass the folks who want to troubleshoot apps to you then. :)
